src/api/middleware/auth.js

```diff
@@ -1,15 +1,30 @@
 const { ErrorTypes } = require("./errorHandler");
+const config = require("../../config/env");
 
 // Middleware to require login in an endpoint
 const authRequired = (req, res, next) => {
     if (req.isAuthenticated()) {
         return next();
     }
     return res.status(401).json({
-        success: false,
         reason: "Must be logged in",
-        code: ErrorTypes.FORBIDDEN,
+        error_code: ErrorTypes.FORBIDDEN,
     });
 };
 
-module.exports = { authRequired };
+// Eventually should be done via a session in an admin account, but at least this will work for now, before a permission system is added
+const isAdmin = (req, res, next) => {
```

Comment:
Basically my idea was as follows: eventually the posts will be placed by companies. As such, it is not relevant to worry much on implementing a permission system, since this was just implemented so that the endpoint is not exposed to anyone who finds it.

Additionally, my idea was that it would not be accessible via the front-end yet, as we can just add the offers manually by using something like `curl`, `httpie` or `postman` for now.

Reply:
Ok, it should be done as part of the CompanyAccounts milestone then

Reply:
So, it should not be included here? Just to be clear on the decision here

Reply:
No, we'll discuss it when implementing the company accounts

Reply:
Resolving this, then!